libs/accounts/recovery-phone/src/lib/recovery-phone.manager.in.spec.ts

```diff
@@ -0,0 +1,79 @@
+import { RecoveryPhoneManager } from './recovery-phone.manager';
+import {
+  AccountDatabase,
+  AccountDbProvider,
+  testAccountDatabaseSetup,
+} from '@fxa/shared/db/mysql/account';
+import { Test } from '@nestjs/testing';
+import { RecoveryPhoneFactory } from './recovery-phone.factories';
+
+describe('RecoveryPhoneManager', () => {
+  let recoveryPhoneManager: RecoveryPhoneManager;
+  let db: AccountDatabase;
+
+  beforeEach(async () => {
+    db = await testAccountDatabaseSetup(['accounts', 'recoveryPhones']);
+    const moduleRef = await Test.createTestingModule({
+      providers: [
+        RecoveryPhoneManager,
+        {
+          provide: AccountDbProvider,
+          useValue: db,
+        },
+      ],
+    }).compile();
+
+    recoveryPhoneManager = moduleRef.get(RecoveryPhoneManager);
+  });
+
+  afterEach(async () => {
+    await db.destroy();
```

Comment:
Do we need to create and destroy the db after each test, or can we just do this once for the test suite.